src/main/java/com/example/event_manager/utils/BillingRaport.java

```diff
@@ -0,0 +1,61 @@
+package com.example.event_manager.utils;
+
+import com.example.event_manager.form.BillingForm;
+import com.thoughtworks.xstream.XStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringReader;
+import javax.xml.transform.Result;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.sax.SAXResult;
+import javax.xml.transform.stream.StreamSource;
+import org.apache.fop.apps.FOPException;
+import org.apache.fop.apps.FOUserAgent;
+import org.apache.fop.apps.Fop;
+import org.apache.fop.apps.FopFactory;
+import org.apache.fop.apps.MimeConstants;
+
+public class BillingRaport {
+
+  private final BillingRaportSchema billingRaportSchema;
+  private String billingsXml;
+  static final String styleXslFile = "src/main/resources/static/billing/newStyle.xsl";
```

Comment:
once you have it as a service, then inject this value as string or resource with @Value annotation https://www.baeldung.com/spring-value-annotation